src/commands/types.d.ts

```diff
@@ -73,3 +73,11 @@ export type NetlifyOptions = {
   state: StateConfig
   frameworksAPIPaths: FrameworksAPIPaths
 }
+
+export interface AddressInUseError extends Error {
```

Comment:
I would just add this into the same error type as optional fields

Reply:
The `Error` type lives in `~external/38d21a27-4378-4895-a56a-d0fa5726a379/lib.es2022.error.d.ts`. I think it's from an dependency/ external source.